src/GUI/ccextractorGUI.c

```diff
@@ -165,6 +166,24 @@ void drop_callback(GLFWwindow* window, int count, const char **paths)
 //	nk_draw_text(canvas, space, "Hardsubs Extraction: Yes", 24, &font->handle, nk_rgb(88, 81, 96), nk_rgb(0, 0, 0));
 //}
 
+char* get_executable_path(){
```

Comment:
This doesn't seem really portable... what about Windows?

Reply:
I don't have a Windows Machine to test it on :(

Reply:
There is a Method for windows too: https://docs.microsoft.com/en-us/windows/desktop/api/libloaderapi/nf-libloaderapi-getmodulefilenamea but i don't have a machine to test it :) If someone volunteers i'll gladly implement it